src/node_perf.h

```diff
@@ -125,6 +126,37 @@ class GCPerformanceEntry : public PerformanceEntry {
   PerformanceGCKind gckind_;
 };
 
+class ELDHistogram : public BaseObject, public Histogram {
+ public:
+  ELDHistogram(Environment* env,
+               Local<Object> wrap,
+               int32_t resolution);
+
+  ~ELDHistogram() override;
+
+  bool RecordDelta();
+  bool Enable();
+  bool Disable();
+  void ResetState() {
+    Reset();
+    exceeds_ = 0;
+    prev_ = 0;
+  }
+  int32_t Exceeds() { return exceeds_; }
+
+  // TODO(jasnell): track the memory used
```

Comment:
I think this could be done somewhat easily in this PR, though `hdr_get_memory_size`?